### DISCUSSION TITLE
Export Cutted/Trimmed Subtitles

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#2064] [Q&A]
I have a video with .ass subtitles. When I export the segments of the video I want, I also want to export the trimmed subtitles (not the full ones) from their respective segments as well. How do I do that?

**Comment by mifi** [comment]
i think this should work:
1. create the segments in losslesscut
2. go to the tracks panel and disable all tracks except subtitles
3. select output format as for example "ass" or "mks"
4. export

**Reply by Jay123210599** [reply]
It works, thanks.
